blueprints/automation/unifiprotect/dynamic_doorbell.yaml

```diff
@@ -0,0 +1,113 @@
+blueprint:
+  name: UniFi Protect Dynamic Doorbell
+  description: |
+    ## UniFi Protect Dynamic Doorbell
+
+    This blueprint will dyanmically update the text on the LCD display if your UniFi Protect Doorbell
```

Comment:
```suggestion
    This blueprint will dynamically update the text on the LCD display if your UniFi Protect Doorbell
```
nit

Reply:
+ it seems that this sentence is not completed? "if your UniFi Protect Doorbell" what? :)